app/src/androidTest/java/ch/epfl/qedit/quiz/QuizActivityTest.java

```diff
@@ -151,4 +151,14 @@ public void testQuizOverviewOnClickDisplayedAgain() {
         onView(withId(R.id.quiz_overview_container)).check(matches(isDisplayed()));
         finishActivity();
     }
+
+    @Test
+    public void testAnswersAreRestored() {
+        launchActivity();
+        onView(withId(R.id.next)).perform(click());
+        onView(withId(R.id.answer_fragment)); /*TODO
```

Comment:
I am just pointing to you that there is a TODO here, maybe you forgot to do something about it ;).

Reply:
That's one of the two tests that I still need to figure out how to do them.